includes/wc-formatting-functions.php

```diff
@@ -574,13 +574,33 @@ function wc_price( $price, $args = array() ) {
 		)
 	);
 
+	$original_price = $price;
+
 	// Convert to float to avoid issues on PHP 8.
 	$price = (float) $price;
 
 	$unformatted_price = $price;
 	$negative          = $price < 0;
-	$price             = apply_filters( 'raw_woocommerce_price', floatval( $negative ? $price * -1 : $price ) );
-	$price             = apply_filters( 'formatted_woocommerce_price', number_format( $price, $args['decimals'], $args['decimal_separator'], $args['thousand_separator'] ), $price, $args['decimals'], $args['decimal_separator'], $args['thousand_separator'] );
+
+	/**
+	 * Filter raw price.
+	 *
+	 * @param float        $raw_price      Raw price.
+	 * @param float|string $original_price Original price as float, or empty string. Since 5.0.0.
+	 */
+	$price = apply_filters( 'raw_woocommerce_price', floatval( $negative ? $price * -1 : $price ), $original_price );
```

Comment:
This extra `floatval()` is redundant because line 580 already converted.

Reply:
Good catch, thanks!